EssentialApp/EssentialApp/ImageCommentViewAdapter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ImageCommentViewAdapter.swift
+//  EssentialApp
+//
+//  Created by Sebastian Vidrea on 09.04.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import EssentialFeed
+import EssentialFeediOS
+
+final class ImageCommentViewAdapter: ResourceView {
+	private weak var controller: ListViewController?
+
+	init(controller: ListViewController? = nil) {
```

Comment:
The controller param shouldn't be optional or have a `nil` default value.

A controller dependency is mandatory at the time of creation:

```suggestion
	init(controller: ListViewController) {
```